backend-comparison/src/burnbenchapp/base.rs

```diff
@@ -16,13 +16,26 @@ use super::{
 };
 
 const FIVE_SECONDS: time::Duration = time::Duration::new(5, 0);
+const USER_BENCHMARK_SERVER_URL: &str = if cfg!(debug_assertions) {
+    // development
+    "http://localhost:8000/benchmarks"
+} else {
+    // production
+    "https://user-benchmark-server-gvtbw64teq-nn.a.run.app/benchmarks"
```

Comment:
I guess this URL will change right?

Reply:
It will not change as long as we keep the same service in Cloud Run.
Anyway this is temporary we will change this to use our domain at some point.